website/docs/faqs/Docs/long-descriptions.md

```diff
@@ -5,19 +5,30 @@ sidebar_label: 'Write long descriptions'
 id: long-descriptions
 ---
 If you need more than a sentence to explain a model, you can:
+1. Split your description over multiple lines using `>`. Interior line breaks are removed and Markdown can be used. This method is recommended for simple, single-paragraph descriptions:
+```yml
```

Comment:
Can you indent the ``yml to line up with the text? I think it would mean 2 spaces?